uint32 saliency_id
time ts
Float32XY screen
Float32XYZ direction
float32 motion
float32 umap
float32 vmap
float32 contrast
float32 confidence
